### ISSUE TITLE
can bar be rendered without padding?

### ISSUE THREAD

**Issue by ShahnazK** [oli107/material-range-bar#92]
i am using range bar in a dialog box, but there is some alignment issue with other components as i am using linear layout. Every other component starts from base line except range bar. It seems there is some internal padding. Is there any way i can remove or reset that padding? please see the attached screen shot
![rangebar](https://cloud.githubusercontent.com/assets/6794083/23341585/694da512-fc70-11e6-85e6-a969bdffa6b5.PNG)

**Comment by krazykira** [comment]
yes this is possible. you just need to set the `custom:rangeBarPaddingBottom` property for rangebar. But keep in mind that  `custom:selectorSize` should never be more than `custom:rangeBarPaddingBottom` otherwise the selectors will be truncated from bottom.

**Comment by aaron861030** [comment]
I have a question related to this. what if I want to remove the top padding? I already set the pin radius to 0 to disable the pin, but it seems the space of it is still accounted in the layout. Anyway we can take that out? Thanks.

**Comment by 2math** [comment]
just set view height to 50dp or something that fits you

**Comment by aaron861030** [comment]
thanks
